iceoryx_posh/test/integrationtests/test_posh_mepoo.cpp

```diff
@@ -20,24 +20,19 @@
 #include "iceoryx_hoofs/internal/units/duration.hpp"
 #include "iceoryx_hoofs/posix_wrapper/timer.hpp"
 #include "iceoryx_hoofs/testing/timing_test.hpp"
+#include "iceoryx_posh/internal/roudi/roudi.hpp"
```

Comment:
I think this one can now be removed. It was only included because some internals were accessed